.changeset/wise-carrots-float.md

```diff
@@ -0,0 +1,15 @@
+---
+'astro': minor
```

Comment:
IDK if that should be considered a major, since we can in theory remove experimental flags at any time

Reply:
Minor seems OK to me!